src/components/TodoList.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from "react";
+import Title from "./Footer";
+
+export default class TodoList extends Component {
```

Comment:
export default goes at the end of the file :smile: Here it would be `export default TodoList;` as the last line of the code.